pkg/target/target.go

```diff
@@ -35,6 +35,8 @@ type Target interface {
 	SeedName() string
 	// ShootName returns the currently targeted shoot cluster name.
 	ShootName() string
+	// ShootControlPlane returns true if shot control plane is targeted.
+	ShootControlPlane() bool
```

Comment:
Why not `ControlPlane`?

Reply:
see comment above. But as I already wrote, and now with some distance to my work, I agree that we should remove the _Shoot_ term. The flag is also not called `--shootcontrolplane` so let's keep it aligned.

Reply:
But we need to add some kind of suffix as we did for the other functions where we added `Name` as it would otherwise clash with the struct names. I decided to call It `ControlPlaneFlag`